### DISCUSSION TITLE
Ethernet switch

### DISCUSSION THREAD

**Discussion by piersandro** [jacklandrin/OnlySwitch#85] [Ideas]
If someone could be interested on a Ethernet switch for your OnlySwitch I created this simple Shell script:

First of wall identificate your Ethernet port name by use this command on your Terminal:
`networksetup -listallnetworkservices`

In my case it is `USB 10/100/1000 LAN`

Open OnlySwitch and select Evolution, add a new shortcut with the Ethernet name and add this commands:

Start with:
`networksetup -setnetworkserviceenabled \"name_of_your_lan_port\" on`

Stop with:
`networksetup -setnetworkserviceenabled \"name_of_your_lan_port\" off`

Check status with:
`networksetup -getnetworkserviceenabled \"name_of_your_lan_port\"`

Output:
`Enabled`

Test all the commands than click Save.

I hope it could be useful.

**Comment by jacklandrin** [comment]
Awesome, I am going to implement a feature that let people can share the evolution they create like shortcuts gallery. People can exchange their idea by it.
